.github/pull_request_template.md

```diff
@@ -18,6 +18,7 @@
 - [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
 - [ ] The developer has manually tested the changes and verified that the changes work.
 - [ ] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).
+- [ ] **For first time contributors**: The developer has ensured all tests have passed **and** the label `ok-to-test` has been added to the PR.
```

Comment:
I don't think it's anyway easy for a new contributor to know which tests should have run, and which have not.
I think this is more knowledge of maintainers.
Suggestion:

```suggestion
- [ ] **For first time contributors**: please reach out to the Reviewers to ensure all tests are being run, ensuring the label `ok-to-test` has been added to the PR.
```

Reply:
Ah yes, they will need to ping reviewers, let me tweak it even more pointing at the owners file, but I like this suggestion, thanks @tarilabs